packages/nimble-components/src/number-field/tests/number-field-matrix.stories.ts

```diff
@@ -76,6 +82,21 @@ export const numberFieldThemeMatrix: StoryFn = createMatrixThemeStory(
     ])
 );
 
+export const numberFieldInteractionsThemeMatrix: StoryFn = createMatrixThemeStory(
+    createMatrix(
+        component,
+        [
+            interactionStates.filter(x => !x[0].includes('Active')), // skip irrelevant active states
+            disabledStates,
+            hideStepStates.filter(x => x[0] !== 'Hide Step'), // always show inc/dec buttons
+            valueStates.filter(x => x[0] !== 'Placeholder'), // don't test placeholder text
+            errorStates.filter(x => x[0] !== 'Error No Message'), // don't test error state w/o error text
```

Comment:
Similar question here. Is there a reason to skip these tests? Is it just to reduce the number of tests being run since the assumption is that testing with the error messages tests everything (and more) that the "Error No Message" would have tested?

Reply:
Correct. Just to reduce noise.